builder/js/ncf.js

```diff
@@ -744,6 +746,20 @@ app.controller('ncf-builder', function ($scope, $modal, $http, $log, $location,
     window.top.location.reload();
   }
 
+  $scope.checkLength = function(str) {
```

Comment:
Not needed anymore  (and would have been better with a checkLength(str,size) !!)